### DISCUSSION TITLE
v1.33.2 chat

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#6531] [Discussion]
Just pinning this so we can temporarily use this as a "chat" for today

**Comment by WorldLanguages** [comment]
As of now the Chrome Web Store is still pending review.

We don't know when exactly Scratch will ship the update. We will not release on Firefox until then. Hopefully we pass the review tomorrow morning on the CWS.

**Comment by WorldLanguages** [comment]
I think this is the first time (in over three years) that we happen to experience a Scratch update like this, where we need to play close attention

**Comment by medlans** [comment]
Will there be a way to change the notification color (I really do not get used to stuff like this as you can tell from that swapped love-it and favorite icon thing)? Also, are they changing the share button again (I feel like I saw something about that but I might be dreaming)

**Comment by Secret-chest** [comment]
> Will there be a way to change the notification color (I really do not get used to stuff like this as you can tell from that swapped love-it and favorite icon thing)? Also, are they changing the share button again (I feel like I saw something about that but I might be dreaming)

Yes please, I don't want white

**Comment by mxmou** [comment]
> Will there be a way to change the notification color (I really do not get used to stuff like this as you can tell from that swapped love-it and favorite icon thing)? Also, are they changing the share button again (I feel like I saw something about that but I might be dreaming)

They only changed the message count on the Messages page. The one in the navigation bar is still orange.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/8d9cedd6-0cc6-4055-9065-9ac78f38815b)
SA will probably have an option to customize these colors in the future: #5602

The share button is exactly the same as before:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/c4280a83-5cd0-4d77-adbb-d775eda6dda1)
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/ead821fe-1967-465f-b690-c58ef6de9320)

**Comment by Hans5958** [comment]
Why an issue? Why not putting this as a discussion (post)?

If not, can the original post be edited so it can also facilitates #5897 and #6115? I think it is nonsensical that three issues of similar topics are pinned at the same time.

**Comment by Samq64** [comment]
> I think this is the first time (in over three years) that we happen to experience a Scratch update like this, where we need to play close attention

There was the studio update.

**Comment by Samq64** [comment]
The update is now live on `scratch-www` pages.

**Comment by BroJac5246** [comment]
> The update is now live on `scratch-www` pages.

And the editor changes have rolled out as well!

**Comment by medlans** [comment]
Oh my god I did not expect to see red when I tried viewing my 24 messages

**Comment by Samq64** [comment]
> Oh my god I did not expect to see red when I tried viewing my 24 messages

I made a suggestion on the Scratch forums to change it: https://scratch.mit.edu/discuss/topic/694217

**Comment by mybearworld** [comment]
On scratchr2 pages too, now.

**Comment by DNin01** [comment]
We were right! Scratch 2 updated to purple as well.

Honestly, I don't mind it!

Still getting used to purple links though.

**Comment by medlans** [comment]
> We were right! Scratch 2 updated to purple as well.
> 
> Honestly, I don't mind it!
> 
> Still getting used to purple links though.

I'm not even on v1.33.2 yet LOL, so it still shows the old default when I have the default colors on
anyway i like grey/blue D:

**Comment by WorldLanguages** [comment]
Version v1.33.2 is now available for 100% of Chrome users
Firefox should start releasing to users in a few minutes

**Comment by medlans** [comment]
> Version v1.33.2 is now available for 100% of Chrome users
> Firefox should start releasing to users in a few minutes

Yea how I wish I could have extensions on ios, anyway I’m gonna check it out later

**Comment by WorldLanguages** [comment]
Looks like the update went smoothly.
We're just receiving some feedback because some small parts of the site don't match the old colors when choosing the default colors preset

**Comment by Samq64** [comment]
While I do think this might have been better as a discussion what's the point of converting this to a discussion a month and a half after it was closed?

**Reply by Hans5958** [reply]
Just for maintenance. Sorry.

**Reply by WorldLanguages** [reply]
More notifications = more activity = more dark green
